guides/_posts/2014-01-5-vfl.html

```diff
@@ -300,34 +301,144 @@
+  footer[right] + [hgap] == ::window[right];
+
+</style>
+
+{% endhighlight %}
+
+If the number of selected elements matches, the position of the elements within each collection of selected
+elements will overlap since you're basically constraining the whole collection to the same values. <a href="http://codepen.io/cbchouinard/pen/yyodWx" target="_blank">Live example.</a>
+
+{% highlight html %}
+
+<section></section>
+<section></section>
+<section></section>
+<article></article>
+<article></article>
+<article></article>
+<footer></footer>
+
+<style type="text/gss">
+
+/* the 3 sections will overlap over each other as the 3 articles. */
+@h |-(section)-(article)-(footer)-| in(::window);
+
+</style>
+
+{% endhighlight %}
+
+Splats will iterate over the collection of selected elements and position elements
+next to each other. <a href="http://codepen.io/cbchouinard/pen/ogerrX" target="_blank">Live example.</a>
+
+{% highlight css %}
+
+<section></section>
+<section></section>
+<article></article>
+<article></article>
+<article></article>
+<div></div>
+<div></div>
+
+<style type="text/gss">
+
+/* elements are not 'pluraly' binded */
+@h |-(section)-10-...-(article)-20-...-(div)...-| in(::window);
+
+</style>
 
-/* Equivalent CCSS */
-#popup[left] + 24 == #find[left];
-#find[right] + [hgap] == #findNext[left];
-#findField[width] >= 30;
-#findNext[right] + [hgap] == #findField[left];
-#findField[right] + 24 == #popup[right];
 {% endhighlight %}
 
+In the previous example, the two `section` will be positioned next to each other with a gap of 10px. The
+three articles will be positioned next to each other with a gap of 20px. Finally the two div will be right next to each
+other without any gap.
+
+<a name="virtuals"></a>
+## Virtual with VFL
+[Virtuals](/guides/constraints) can be created and/or positionned using VFL expressions.
 
+You can position virtual as any other elements :
 
-<h2>VFL Chains</h2>
+{% highlight css %}
 
-Chains are a powerful mechanism to take each element within an VFL alignment (excluding a containment element & points) and construct a chained constraint statement against a property of each element.  
+/* horizontally align two virtuals with a gap of 10px between each
+@h |-("area1")-10-("area2")-| in(::window)
 
-Chains take the form `chain-{Property}({Head Operator} {Expression} {TailOperator})`.  
+{% endhighlight %}
+
+<a name="rulesets"></a>
+## VFL rulesets
+
+You can use a VFL expression as the rule of a [rulesets](/guides/constraints/#rulesets). It will
+behave in the same way as a selector rule. GSS will create a collection of all elements involved
+in the VFL expression and iterates over them.
+
+This allows you to do interesing things. For example, if you want all the elements of the VFL expression
+to have the same width you can do the following:
+
+{% highlight css %}
+
+@h |-(section)-10-...-(article)-20-...-(div)...-| in(::window) {
+      width: == :next[width];
```

Comment:
Not sure we should be promoting the use of `:next` here for a few reasons:
* couples the layout to the DOM
* the next element in the DOM may not be the next element in the VFL statement
* the same result can be achieved via something like `width: == [item-width];`

This is based on my understanding that `:next` means "the next sibling element in the DOM".

I'm not yet aware of any special handling around `:next` that addresses these concerns but would like to see a section in the docs that explains what happens in these scenarios.